app-frontend/src/app/core/services/histogram.service.js

```diff
@@ -0,0 +1,20 @@
+export default (app) => {
+    class HistogramService {
+        constructor($http, authService) {
+            'ngInject';
+            this.authService = authService;
+            this.$http = $http;
+        }
+
+        getHistogram(project, scenes) {
+            return this.$http.post(
+                `/tiles/${project}/histogram/?token=${this.authService.token()}`,
```

Comment:
This needs to use the tile server location otherwise it's going to go back to the API server which does not have this route.